resources/assets/ckeditor/plugins/filelink/plugin.js

```diff
@@ -9,26 +8,22 @@ CKEDITOR.plugins.add( 'filelink', {
         });
 
         var cmd = editor.addCommand('filelink.cmd', {exec:showDialogPlugin});
-
-        CKEDITOR.dialog.addIframe(
-            'filelink.dlg',
-            'File Link',
-            editor.config.fileLinkUploadUrl,
-            800,
-            400,
-            function(){
-            }
-        );
     }
 });
 
 function showDialogPlugin(e){
-    e.openDialog('filelink.dlg');
+    kQuery.magnificPopup.open({
```

Comment:
@ercanozkaya I replaced the iframedialog plugin with manificPopup, everything is fine except that the Insert `insert-link` event in FILElink still cannot close the modal, same with Cancel `insert-modal-cancel`. `window.parent.SqueezeBox` within FILElink view looks okay but seems to belong to modal other than the one we opened up here.

![image](https://user-images.githubusercontent.com/5572579/34191452-1be0234e-e582-11e7-8ef5-1015d5e9fcfd.png)

Reply:
@jebbdomingo We are fixing that as part of joomlatools/fileman#353 so it won't be an issue.